recipes/vulkan-utility-libraries/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,14 @@
+#include <vulkan/layer/vk_layer_settings.hpp>
+#include <iostream>
+
+int main() {
+#ifdef USE_PRE_268_API
+    VlLayerSettingSet layerSettingSet = VK_NULL_HANDLE;
+    vlCreateLayerSettingSet("VK_LAYER_LUNARG_conan", nullptr, nullptr, nullptr, &layerSettingSet);
+#else
+    VkuLayerSettingSet layerSettingSet = VK_NULL_HANDLE;
+    vkuCreateLayerSettingSet("VK_LAYER_LUNARG_conan", nullptr, nullptr, nullptr, &layerSettingSet);
+#endif
```

Comment:
```suggestion
    VkuLayerSettingSet layerSettingSet = VK_NULL_HANDLE;
    vkuCreateLayerSettingSet("VK_LAYER_LUNARG_conan", nullptr, nullptr, nullptr, &layerSettingSet);
```
When adopting only 1.3.296 version.